okrs_2019.md

```diff
@@ -0,0 +1,209 @@
+
+
+## SourceCred Community Growth
+
+We envision SourceCred being a thriving open-source community which is
+passionate about recognizing and rewarding open-source contributors, about the
+art of software engineering itself, and, of course, about building SourceCred.
+
+Right now we have a pretty good start in that the founders are passionate about
+these things ;) but we hope to grow the engaged community of contributors and
+users.
+
+On the initiative side, we want to improve the documentation around all parts of
+SourceCred so that the project is easier to get up to speed on. We also intend
+to produce content explaining SourceCred, like blog posts and video talks.
+
+### Initiatives
+- * [ ] Improve SourceCred's documentation
+  - * [ ] Document the overall vision, mission, and philosophy
+  - * [ ] Document the overall technical architecture
+  - * [ ] Document the attribution algorithm (e.g. by writing a whitepaper)
+  - * [ ] Document the plugin system, and how to add a new one
+  - * [ ] Add API docs
+- * [ ] Produce engaging blog posts
+- * [ ] Produce engaging audiovisual content
+
+### Metrics
+
+| Metric                                  | Goal | Q4 | Q1 | Q2 | Q3 | Q4 |
+| --------------------------------------- | ---- | -- | -- | -- | -- | -- |
+| # of contributors w/ >10 cred           | 15   | 2  |    |    |    |    |
+| # of unique code committers             | 30   | 3  |    |    |    |    |
+| # of core team members                  | 3    | 1  |    |    |    |    |
+| # of Discord community members          | 500  | 61 |    |    |    |    |
+| # of documentation goals accomplished   | 5    | 0  |    |    |    |    |
+
+At the moment, we aren't including metrics around engagement with the
+explanatory content we produce (tracking the number of new members joining the
+community is a good measurement of what we acutally care about, namely
+engagement with the project). However, if we decide to put more emphasis on
+content production, we might update these metrics.
+
+## SourceCred Adoption
+
+Right now, SourceCred is a prototype in search of some real users. Actual users
+will be a great source of feedback on the quality of our attributions, and also
+will bring more people into our community.
+
+### Initiatives
+
+One initiative we’re quite excited about is a planned integration with
+[OpenCollective]. We intend to embed SourceCred results into every open-source
+collective’s page (e.g. here is the [page for Webpack]). This way, collectives
+can thank their leading contributors as well as their financial sponsors. We
+have enthusiastic buy-in from the OpenCollective team, and think this could be
+the start of a great partnership, in addition to bringing more attention and
+users to SourceCred.
+
+[OpenCollective]: https://opencollective.com/
+[page for Webpack]: https://opencollective.com/webpack#contributors
+
+- [ ] Integrate with OpenCollective ([#935])
```

Comment:
> Integrate with OpenCollective

Not listed as a requirement: some hosting on our part. Is it expected
that Open Collective will be managing all of this (maintaining
SourceCred instances, keeping them up to date, serving recent
attributions, …?)

Not listed as a requirement: better stability in the Mirror module.
Hardcoded blacklisting won’t cut it. Probably our support for deletions
won’t cut it either. As a quick fix we can always include some logic to
throw away the cache if the Mirror module gives one of these errors, but
that won’t be fun for (a) whoever pays for hosting and (b) anyone who
wants to see quickly updated attributions.

Reply:
Yeah, we'll need to figure out hosting. I think initially a cron style job on AWS that pushes json data onto GitHub pages will suffice.

In the spirit of "this is a living document", I am going to spec out the work for this once we're to implementing it.